state-chain/runtime/src/lib.rs

```diff
@@ -1600,6 +1600,39 @@ impl_runtime_apis! {
 					(asset, pallet_cf_lp::FreeBalances::<Runtime>::get(&account_id, asset).unwrap_or(0))
 				).collect(),
 				earned_fees: AssetMap::from_iter(pallet_cf_lp::HistoricalEarnedFees::<Runtime>::iter_prefix(&account_id)),
+				boost_balances: AssetMap::from_iter(Asset::all().map(|asset| {
```

Comment:
This can be condensed a little, AssetMap has a constructor for this: 

```suggestion
				boost_balances: AssetMap::from_fn(|asset| {
```